examples/complete/variables.tf

```diff
@@ -16,14 +20,37 @@ variable "prefix" {
   default     = "complete"
 }
 
-variable "resource_group" {
-  type        = string
-  description = "An existing resource group name to use for this example, if unset a new resource group will be created"
-  default     = null
-}
-
 variable "resource_tags" {
   type        = list(string)
   description = "Optional list of tags to be added to created resources"
   default     = []
 }
+
+variable "region" {
+  type        = string
+  description = "IBM Cloud region where event notification will be created, supported regions are: us-south (Dallas), eu-gb (London), eu-de (Frankfurt), au-syd (Sydney), eu-es (Madrid)"
+  default     = "us-south"
+}
+
+variable "kms_region" {
```

Comment:
remove this (see https://github.com/terraform-ibm-modules/terraform-ibm-event-notifications/pull/52/files#r1388247823)

Reply:
Removed.